azure-pipelines/maven-release/msal-maven-release.yml

```diff
@@ -18,7 +18,7 @@ resources:
   - repository: common
     type: github
     name: AzureAD/microsoft-authentication-library-common-for-android
-    ref: pedroro/release-pipelines
+    ref: dev
```

Comment:
At some point we should investigate if we can move away from this model and just reference the templates from the submodule

Reply:
Yes, that is what I would like to achieve at some point, but I have not had the time to investigate and do it this way